.github/workflows/python-publish.yml

```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   pypi-publish:
     if: |
+      github.repository_owner == 'oauthlib' &&
```

Comment:
is the owner right permission? or it should be member or something?

Reply:
Yes, see the docs:

> The repository owner's username. For example, `octocat`.

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

And see this similar PR: https://github.com/python-attrs/attrs/pull/1179